### ISSUE TITLE
Publish the changelog, release notes

### ISSUE THREAD

**Issue by YevheniiSemendiak** [neuro-inc/platform-docs#9]
Problem: the channels where our users get platform updates are closed for them.
Currently, there's only a semi-closed Slack channel (meaning that one needs to be approved to join this channel).

We can start with publishing the changelog of platform clients (SDK, CLI, flow, extras web).
Even since it's already being gathered in changelogs https://github.com/neuro-inc/neuro-all/blob/master/CHANGELOG.md

I'm thinking of a dedicated page in docs, divided into several subpages:
- `neuro-all` https://github.com/neuro-inc/neuro-all/blob/master/CHANGELOG.md
- `project template` https://github.com/neuro-inc/cookiecutter-neuro-project/blob/master/CHANGELOG.md
- `platform web`, should be added
- `base image` should be added

Origin: https://github.com/neuro-inc/platform-web/issues/163

**Comment by YevheniiSemendiak** [comment]
As a step further, we could do smth like W&B does in our WebApp:
<img width="204" alt="image" src="https://user-images.githubusercontent.com/32543098/167171570-b738d124-2788-4a4d-a58b-7c3e671d2b1a.png">

But this is for the later discussion.

**Comment by asvetlov** [comment]
I believe just a page with links to https://github.com/neuro-inc/neuro-all/blob/master/CHANGELOG.md,  https://github.com/neuro-inc/cookiecutter-neuro-project/blob/master/CHANGELOG.md, etc. is enough.
There is no need to parse these resources and re-render data again.
At least not on the current development stage.

**Comment by YevheniiSemendiak** [comment]
Agreed, according to our [plans](https://docs.google.com/spreadsheets/d/1dZpmVtVN-BMmq0InW7Cmp84OFDDchqN5kXe-ux-LV3c/edit#gid=0) this is to be done in phase 4, but I believe our backlog for docs in phases 1-3 is not yet full. @eviltwinhans , MB we put it into the pase 1 or 2?

**Comment by eviltwinhans** [comment]
2do: transfer to no-code, make it a story, assign to Sasha
